src/components/atoms/Timeline/ScaleList.tsx

```diff
@@ -0,0 +1,117 @@
+import { memo } from "react";
+
+import { styled } from "@reearth/theme";
+import { XXSRegular } from "@reearth/theme/fonts";
+
+import { EPOCH_SEC, STRONG_SCALE_WIDTH, NORMAL_SCALE_WIDTH, PADDING_HORIZONTAL } from "./constants";
+
+type Props = {
+  gapHorizontal: number;
+} & ScaleListInnerProps;
+
+export const ScaleList: React.FC<Props> = ({ gapHorizontal, ...rest }) => {
+  return (
+    <ScaleContainer style={{ gap: `0 ${gapHorizontal}px` }}>
+      <ScaleListInner {...rest} />
+    </ScaleContainer>
+  );
+};
+
+const MONTH_LABEL_LIST = [
+  "Jan",
+  "Feb",
+  "Mar",
+  "Apr",
+  "May",
+  "Jun",
+  "Jul",
+  "Aug",
+  "Sep",
+  "Oct",
+  "Nov",
+  "Dec",
+];
+
+type ScaleListInnerProps = {
+  start: Date;
+  scaleCount: number;
+  hoursCount: number;
+  scaleInterval: number;
+  strongScaleHours: number;
+};
+
+const ScaleListInner: React.FC<ScaleListInnerProps> = memo(function ScaleListPresenter({
+  start,
+  scaleCount,
+  hoursCount,
+  scaleInterval,
+  strongScaleHours,
+}) {
+  const strongHours = hoursCount * strongScaleHours;
+  const lastStrongScaleIdx = strongHours * (scaleCount / strongHours);
+  return (
+    <>
+      {[...Array(scaleCount + 1)].map((_, idx) => {
+        const isHour = idx % hoursCount === 0;
+        const isStrongScale = idx % strongHours === 0;
+        if (isStrongScale && idx !== lastStrongScaleIdx) {
+          const d = new Date(start.getTime() + idx * EPOCH_SEC * scaleInterval);
+          const year = d.getFullYear();
+          const month = MONTH_LABEL_LIST[d.getMonth()];
+          const date = `${d.getDate()}`.padStart(2, "0");
+          const hour = `${d.getHours()}`.padStart(2, "0");
+          const label = `${month} ${date} ${year} ${hour}:00:00.0000`;
+
+          return (
+            <LabeledScale key={`${idx}`}>
```

Comment:
Is there a reason you are writing it like `key={`${idx}`}` and not just `key={idx}`?

Reply:
If you are converting it to a string, is there a reason you are?

Reply:
This is because, I often use [this jsx-key eslint rule](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-key.md) but in reearth this rule is disabled so I fixed to use string.
[refactor: use number idx as key](https://github.com/reearth/reearth-web/pull/264/commits/400fd05e820310184d4c5f4564fa52904059f338)

Reply:
But maybe we can try to enable this eslint rule.